rilldata/rill
# Run profile queries after fetching columns

## Checklist

## Summary
#### Issue addressed: 
closes #2662

#### Details:


## Steps to Verify
1. Create a model from any source.
2. Add `select <column1> from <source>`
3. Replace the `column1` from `column2` by pasting and not typing. This way at a single shot the column is replaced.
4. Doing this back and forth should not add any 400 errors for profiling queries.
